index.html

```diff
@@ -2,10 +2,82 @@
 <html lang="ru-BLR">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>What do we offer</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+<style>
```

Comment:
@Hashlemon it's better to transfer styles into css file

Reply:
@Hashlemon Also. As I remember right, Roboto is an inbuilt font

Reply:
Done. 
Roboto is not inbuilt in Ubuntu at least. So I leave this font taken outside